src/ResourceManager/Network/Commands.Network/ApplicationGateway/NewAzureApplicationGatewayCommand.cs

```diff
@@ -83,6 +83,12 @@ public class NewAzureApplicationGatewayCommand : ApplicationGatewayBaseCmdlet
              HelpMessage = "The list of authentication certificates")]
         public List<PSApplicationGatewayAuthenticationCertificate> AuthenticationCertificates { get; set; }
 
+        [Parameter(
+             Mandatory = false,
+             ValueFromPipelineByPropertyName = true,
+             HelpMessage = "The list of trusted root certificates")]
+        public List<PSApplicationGatewayTrustedRootCertificate> TrustedRootCertificates { get; set; }
```

Comment:
why is this a plural? the param name should be singular i.e `TrustedRootCertificate`. [Documentation](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/azure-powershell-design-guidelines.md#noun-singularity).